src/porepy/models/constitutive_laws.py

```diff
@@ -431,7 +431,7 @@ class FluidDensityFromPressure:
 
     """
 
-    def fluid_compressibility(self, subdomains: list[pp.Grid]) -> pp.ad.Scalar:
```

Comment:
Shouldn't we keep it as specific as possible?

Reply:
I agree. But this means that we should change all the signatures of the methods belonging to `ConstantX`-type mixin classes. Since this method was the only one that returned a `pp.ad.Scalar`, I assumed there were other reasons for using `pp.ad.Operator` in the other methods.

I'm OK with either option as long as we are consistent. What do you think?

Reply:
You're right. I think the idea of using Operator was to indicate that the methods may be extended/overridden to return an Operator. Opinions, @keileg? What's the most principled solution?

Reply:
I recall the same rationale that you indicate: To allow for general inheritance, the return type should be an operator.